crates/cadenza-eval/src/generated/snapshots/cadenza_eval__generated__test_data__fn_match_phi__fn_match_phi_ir.snap

```diff
@@ -0,0 +1,5 @@
+---
+source: crates/cadenza-eval/src/generated/test_data.rs
+expression: "t::ir(\"# Test function with match that generates phi nodes\\n\\nfn abs x =\\n    match x > 0\\n        (true -> x)\\n        (false -> 0 - x)\\n\\nabs 5\\nabs (-3)\\n\")"
+---
+# IR Module
```

Comment:
It doesn't look like any IR is actually getting generated for this module. Which makes it difficult to tell if the change we just made is actually correct.